Makefile

```diff
@@ -91,3 +91,6 @@ ifneq ($(wildcard /.dockerenv), )
 endif
 
 validate: fmt
+
+uninstall:
+	$(foreach file,containerd containerd-shim ctr,rm -rf /usr/local/bin/$(file);)
```

Comment:
Just to be safe we don't need the `-r` on the `rm`.  Do you think we need the `-f` as well?